ext/gd/libgd/gd.h

```diff
@@ -50,7 +50,7 @@ extern "C" {
 	pixels are represented by integers, which
 	must be 32 bits wide or more.
 
-	True colors are repsented as follows:
+	True colors are represented as follows:
```

Comment:
"Just" for typos, I do not object not upstreaming first (this one [is still present](   True colors are repsented as follows:), not the other), but would be ideal if they both match to not increase differences further.

Reply:
https://github.com/libgd/libgd/pull/954